plugin/timeouts/README.md

```diff
@@ -0,0 +1,58 @@
+# timeouts
+
+## Name
+
+*timeouts* - allows you to configure the server read, write and idle timeouts for the TCP, TLS and DoH servers.
+
+## Description
+
+CoreDNS is configured with sensible timeouts for server connections by default. However in some cases
+for example where CoreDNS is serving over a slow mobile data connection the default timeouts are not
+optimal.
+
+Additionally some routers hold open connections when using DNS over TLS or DNS over HTTPS. Allowing
+a longer idle timeout helps performance and reduces issues with such routers.
+
+The *timeouts* "plugin" allows you to configure CoreDNS server read, write and idle timeouts.
+
+## Syntax
+
+~~~ txt
+timeouts READ [WRITE] [IDLE]
```

Comment:
I would choose different approach to passing timeout durations, your approach forces me to provide all configurations when I want to override just IDLE timeout, so I would go with extra knobs configuration

```suggestion
timeouts {
	read DURATION
	write DURATION
	idle DURATION
}
```
=> If I want to override just write timeout I can do
```suggestion
timeouts {
	write 5s
}
```
if I want to override just idle timeout I can do 
```suggestion
timeouts {
	idle 5s
}
```

Reply:
This makes a lot more sense. I am not entirely sure why I didn't do this to start with.